src/app/dashboard/executions/executions.component.html

```diff
@@ -29,7 +29,12 @@ <h3>{{ 'allexecutions' | translate }}</h3>
             {{execution.metisPlugins[execution.currentPlugin].pluginType}}
           </span>
         </td>
-        <td class="nowrap">{{execution.metisPlugins[execution.currentPlugin] ? execution.metisPlugins[execution.currentPlugin].executionProgress.processedRecords : '-'}}</td>
+        <td class="nowrap">
+          {{execution.metisPlugins[execution.currentPlugin] ? execution.metisPlugins[execution.currentPlugin].executionProgress.processedRecords - execution.metisPlugins[execution.currentPlugin].executionProgress.errors : '-'}} 
```

Comment:
As discussed, we can use a pipe operator here to obtain the count of successful/failed records from the plugin.